content_scripts/vimium_frontend.coffee

```diff
@@ -229,10 +235,13 @@ getActiveState = ->
 #
 # The backend needs to know which frame has focus.
 #
-registerFocus = ->
-  # settings may have changed since the frame last had focus
-  settings.load()
-  chrome.runtime.sendMessage({ handler: "frameFocused", frameId: frameId })
+registerFocus = (event) ->
+  if event.target == window
+    # Settings may have changed since the frame last had focus.
+    settings.load()
+    # Don't register frameset containers; focusing them is no use.
+    unless document.body?.tagName.toLowerCase() == "frameset"
```

Comment:
I believe that it's focused when you drag the boundary between two frames to resize them, and probably also if you click the frames' borders.

> we end up registering the frame set on `focus`, and it becomes part of the `gf` cycle.

If they *are* focusable, we should probably register them as normal and just mark them as `<frameset>`s, so we can switch out of them/keep track of our position reliably, but can still skip them when switching frames with `gf`.